### ISSUE TITLE
Transition to compiled documentation

### ISSUE THREAD

**Issue by CollinHeist** [TitleCardMaker/TitleCardMaker#339]
I should have done this a long time ago, but the project documentation should move to compiled documentation. This gives more versatility with not just basic Markdown elements (e.g. collapsibles, admonitions, etc.).

I'm thinking `mkdocs` is the best option, and hosted on a GitHub site (although I don't have experience with this).

Perhaps this should be timed with the v2.0 release of the web UI, as all the existing documentation will need to be migrated.

# Outstanding Tasks

- [ ] Document changes between v1 -> v2
- [ ] Finish tutorial
   - [ ] Local Docker install / setup
   - [ ] Sync from Emby
   - [ ] Sync from Plex
   - [ ] Sync from Jellyfin
   - [ ] Create "basics" page that describes what some common terms are

**Comment by CollinHeist** [comment]
New doc site is https://titlecardmaker.readthedocs.io/

**Comment by mchangrh** [comment]
If you don't like readthedocs and would like to stick to github pages, it would be trivial to set up mkdocs with github actions.

**Comment by CollinHeist** [comment]
@mchangrh Is the main difference there the `.github.io` domain? Or is there more of a difference. I don't mind RTD, but I'm also not opposed to a normal GitHub page, I just already had a RTD workflow set up

**Comment by mchangrh** [comment]
RTD is not a part of GitHub, it runs its own infrastructure, has their own business model and is managed independently of GH.

**Comment by CollinHeist** [comment]
Yeah I know that, I mean is there a content or workflow difference between the two?

**Comment by mchangrh** [comment]
aside from hosting platform, there should not be a significant difference between the two.

**Comment by CollinHeist** [comment]
Okay that's good to know. I think I'll stick with RTD for now, and maybe move in the future! It's nice to know switching would be easy
